concourse/tasks/gatling-workflow-load-test.yml

```diff
@@ -0,0 +1,36 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: maven
+    tag: 3.6.3-ibmjava-8-alpine
+inputs:
+  - name: git-master
+outputs:
+  - name: gatling-results
+    path: git-master/loadtests/target/gatling
+run:
+  path: /bin/bash
+  args:
+    - -euo
+    - pipefail
+    - -c
+    # User journey is 16 requests long, and we want close to 1000 req/s, so inject 1000/16 users/s here (63),
+    # with a pause of 1 second between requests.
+    # Run simulation for 30 mins (1800s) to ensure workflow execution time is covered.
+    # Suppress html file output (doesn't seem possible to completely remove file output...)
+    # Suppress console log to final report only (done by turning off console writer)
+    - |
+      cd loadtests
+      mvn gatling:test \
+        -DinjectUsersPerSecond=36 \
+        -DinjectDurationSeconds=1200 \
+        -DpauseBetweenRequestsInSeconds=1 \
+        -DnumberOfRepetitions=1 \
+        -Dgatling.simulationClass=svp.SvpSimulationConstantUsersPerSec \
```

Comment:
i don't see the following scala class in the PR anymore SvpSimulationConstantUsersPerSec.scala?

Reply:
ah balls, must have lost it in the rebase. I'll reinstate it, thanks!

Reply:
done, thanks for the catch!